FIRE_DATA_API/fire_data_api/app.py

```diff
@@ -14,9 +14,10 @@
 from flask import Flask, jsonify, request, json
 from flask_restful import Api, reqparse
 from flask_cors import CORS
+import requests
 
 # local imports
-from .functions import fires_list, haversine, sort_fires, fires_list_type
+from .functions import fires_list, haversine, sort_fires, fires_list_type,get_aqi_data,get_nearest_stations
```

Comment:
Shorten this line to < 79 characters in accordance to PEP8 guideline.

Reply:
shortened the line